sklearn/utils/validation.py

```diff
@@ -218,11 +218,12 @@ def _ensure_sparse_format(spmatrix, accept_sparse, dtype, copy,
     spmatrix : scipy sparse matrix
         Input to validate and convert.
 
-    accept_sparse : string, list of string or None (default=None)
+    accept_sparse : string, list of strings or boolean (default=None)
```

Comment:
"string, boolean or list of strings" make it clearer. You avoid the potential confusion of reading list of strings or list of booleans.